bin/resources/local/rspec_command

```diff
@@ -85,8 +85,15 @@ rspec_path = File.absolute_path(File.join(__FILE__, './../../../../gems/bin/rspe
 ::ENV['OS_SERVER_DATABASE_NAME'] = 'os_local'
 ::ENV['OS_SERVER_LOG_PATH'] = File.join(output_dir, 'logs', 'rails.log')
 ::ENV['RAILS_ENV'] = 'local-test'
-sys_call = " \"#{ruby_exe}\" \"#{bundle_path}\" exec \"#{ruby_exe}\" \"#{rspec_path}\" --tag ~depends_r --tag ~depends_gecko --tag ~depends_resque --exclude-pattern "\
+
+#If this is a travis build and it's BUILD_TYPE=test we want to omit the openstudio_algo_spec.rb tests
+if ENV['BUILD_TYPE'] == 'test'
+  sys_call = " \"#{ruby_exe}\" \"#{bundle_path}\" exec \"#{ruby_exe}\" \"#{rspec_path}\" --tag ~depends_r --tag ~depends_gecko --tag ~depends_resque --exclude-pattern "\
+  "\"**/models/cluster_spec.rb\" --exclude-pattern \"**/features/openstudio_algo_spec.rb\""
+else
+  sys_call = " \"#{ruby_exe}\" \"#{bundle_path}\" exec \"#{ruby_exe}\" \"#{rspec_path}\" --tag ~depends_r --tag ~depends_gecko --tag ~depends_resque --exclude-pattern "\
   "\"**/models/cluster_spec.rb\""
+end 
 puts "System call will be: '#{sys_call}' in directory '#{Dir.pwd}'" if options[:debug]
```

Comment:
I just added this condition to exclude the features/openstudio_algo_spec.rb for local builds when running on Travis. I saw we were already doing this to exclude the /models/cluster_spec.rb so that is why I added it here. This is probably not the best way to exclude certain tests based on job build type.  Unless we can include rspec tag markers for the build type.  This does work right now